x-pack/plugin/eql/src/main/antlr/EqlBase.g4

```diff
@@ -205,12 +205,29 @@ LP: '(';
 RP: ')';
 PIPE: '|';
 
+fragment STRING_ESCAPE
+    : '\\' [btnfr"'\\]
```

Comment:
you may want to add `u` here to get a little more protection. I think the handleUnicodeEscaoe assumes this is well formed and could incorrectly accept cases like `\u1fd04}`
```suggestion
    : '\\' [btnfru"'\\]
```

Reply:
`\u` is here: https://github.com/elastic/elasticsearch/pull/70514/files#diff-15260c68483eba5f9c6751c895d68b27dd571f82598da666194d9e8c5c4757d7R217,
so it's only allowed with the curly braces,  please see the tests: https://github.com/elastic/elasticsearch/pull/70514/files#diff-9b0fe07d974e767795fe949863d7ae0b9d13787854b1435cba514335b75e62e5R215

Reply:
oh you're totally right, sorry.
I was thinking this was an inverter character class for a second. I do something like that -- accept more input from antlr and then reject in the Visitor. You're good

Reply:
No worries! Thx!